Api/Endpoints/Webhooks.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace Lokalise\Endpoints;
+
+use \Lokalise\LokaliseApiResponse;
+use \Lokalise\Exceptions\LokaliseApiException;
+use \Lokalise\Exceptions\LokaliseResponseException;
+
+/**
+ * Class Webhooks
+ * @package Lokalise\Endpoints
+ * @link https://lokalise.co/api2docs/php/#resource-webhooks
+ */
+class Webhooks extends Endpoint implements EndpointInterface
+{
+
+    /**
+     * @link https://lokalise.co/api2docs/php/#transition-list-all-webhooks-get
```

Comment:
All the links in this files PHPDocs are from `translation` and with outdated url's